### PR TITLE
🎨 Feature: Connect Wallets to backend

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?

This PR adapts the frontend to [the current backend models](https://github.com/ITISFoundation/osparc-simcore/pull/4565) and connects to the backend.

Wallets

## Related issue/s

related to https://github.com/ITISFoundation/osparc-issues/issues/922
fixes https://github.com/ITISFoundation/osparc-issues/issues/1050

### CODE REVIEW

**Review by matusdrobuliak66 — APPROVED** [review]
really nice :tada:

File: services/static-webserver/client/source/class/osparc/data/model/Wallet.js [hunk 1]
```diff
@@ -22,14 +22,14 @@ qx.Class.define("osparc.data.model.Wallet", {
...
       name: walletData["name"],
       description: walletData["description"] ? walletData["description"] : null,
       thumbnail: walletData["thumbnail"] ? walletData["thumbnail"] : null,
-      walletType: walletData["type"] ? walletData["type"] : "personal",
-      accessRights: walletData["accessRights"],
-      credits: walletData["credits"] && walletData["credits"]["left"] ? walletData["credits"]["left"] : 0,
-      active: walletData["active"] ? walletData["active"] : false
+      owner: walletData["owner"] ? walletData["owner"] : null,
+      status: walletData["status"] ? walletData["status"] : false,
+      creditsAvailable: walletData["available_credits"] ? walletData["available_credits"] : 20,
```
Comment (sanderegg) [inline services/static-webserver/client/source/class/osparc/data/model/Wallet.js:31]: not weird to have 20 credits when none available?

**Review by sanderegg — APPROVED** [review]
very nice!

File: services/static-webserver/client/source/class/osparc/data/model/Wallet.js (hunk 1 above)
Comment (GitHK) [reply services/static-webserver/client/source/class/osparc/data/model/Wallet.js:31]: I would have this in the backend do avoid "hackers" which can give themselves more credits

**Pull request merged**